src/alchemlyb/tests/parsing/test_gomc.py

```diff
@@ -30,3 +30,16 @@ def test_u_nk():
         assert u_nk.index.names == ['time', 'Coulomb-lambda', 'VDW-lambda']
         assert u_nk.shape == (1000, 23)
 
+def test_extract():
+    """Test that u_nk and dHdl have the correct form when extracted from files.
+
+    """
+    dataset = load_benzene()
+
+    for filename in dataset['data']:
```

Comment:
Just do this on a single file should be fine. No need to go through all the files.

Reply:
You are right :-)